pkg/cluster/kubernetes/policies_test.go

```diff
@@ -186,13 +186,15 @@ func TestUpdatePolicies(t *testing.T) {
 		},
 	} {
 		t.Run(c.name, func(t *testing.T) {
-			caseIn := templToString(t, annotationsTemplate, c.in)
-			caseOut := templToString(t, annotationsTemplate, c.out)
+			cLocal := c // Use copy to avoid races between parallel tests
```

Comment:
It's not a race between tests you are worried about here, it's between each test and the thread in which the loop is run.

Reply:
Indeed, it's the loop which modifies the variable. That's what I actually meant, but I phrased it wrongly. Will fix (for #2681 as well).